res/css/views/elements/_InteractiveTooltip.scss

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 .mx_InteractiveTooltip_wrapper {
     position: fixed;
-    z-index: 5000;
+    z-index: 4000;
```

Comment:
4000 is the same as the Dialog wrapper, ideally one is inherently above the other and not at the same z-index

Reply:
ohh i understand, i tried to place the it's z-index a little above the dialog wrapper like 4001 or so, it's not hiding the dropdown behind the search modal even when it has higher z-index than both, that's why i made it same. Or should i just remove the z-index, it still works when i do that. i don't know what do you think?

Reply:
Got it, Thanks